pyfar/dsp/dsp.py

```diff
@@ -1438,7 +1438,7 @@ def time_shift(signal, shift, unit='samples'):
     return shifted.reshape(signal.cshape)
 
 
-def deconvolve(system_output, system_input, fft_length=None, **kwargs):
+def deconvolve(system_output, system_input, freq_range=None, fft_length=None, **kwargs):
```

Comment:
It would usually be bad style to change the order of arguments, but in this case I'm also in favor of it. Any other thoughts?

Reply:
I not sure, if it is worth do break backwards compatibility here, unless we do not provide a default. Using the keywords allows an arbitrary order anyway... Any other opinions?
```suggestion
def deconvolve(system_output, system_input, fft_length=None, freq_range=None, **kwargs):
```

Reply:
I'll have to agree with Simon here, please do not change the order of input arguments for existing functions. It breaks backwards compatibility. Prior to this PR, freq_range could have been passed as part of kwargs.